pkg/podman/podman.go

```diff
@@ -144,6 +146,36 @@ func (p *Podman) Copy(id, src, dest string) error {
 	return nil
 }
 
+// Inspect returns the sha256 digest for a given container image on a specific architecture
+func (p *Podman) Inspect(img string, arch string) (string, error) {
```

Comment:
This function should be called `Digest()` and not `Inspect()` to properly reflect what it is doing.